samples/sensor/adt7420/boards/frdm_k64f.overlay

```diff
@@ -4,13 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-&i2c1 {
+&i2c0 {
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_STANDARD>;
-	adt7420@13 {
+	adt7420@48 {
 		compatible = "adi,adt7420";
-		reg = <0x13>;
+		reg = <0x48>;
 		label = "ADT7420";
-		int-gpios = <&gpioc 6 0>;
+		int-gpios = <&gpioc 16 0>;
```

Comment:
Let's replace with `<&arduino_header 6 0> /* D0 */`.

Reply:
Makes sense, but this propagated from the original conversion to devicetree, and isn't specific to this sample: there are at least twelve samples that bypass the Ardunio alias and nexus map for this platform, probably more that do it for other platforms.

Can we do this in a separate PR that fixes it for all samples at once, after the dust settles on topic-gpio and it won't create conflicts?

Reply:
Sure